src/appleseed.shaders/src/appleseed/as_subsurface.osl

```diff
@@ -92,13 +92,13 @@ shader as_subsurface
         string label = "Depth Scale",
         string page = "Subsurface"
     ]],
-    int in_sss_maximum_ray_depth = 2
+    int in_sss_maximum_ray_depth = 100
     [[
         string as_maya_attribute_name = "sssMaximumRayDepth",
         string as_maya_attribute_short_name = "ssd",
         int min = 1,
```

Comment:
min = 1 is intentional here I presume?

Reply:
I'm not entirely sure if to allow a minimum of 0 since that disables SSS completely and this is an exclusively SSS shader (with a specular on top). What do you think? Allow 0 or always keep at least 1 bounce for the SSS to produce anything?

Reply:
It actually looks like this parameter isn't used at all?

Reply:
You're right,  i added it in stdsurface but didn't in this PR. I have it in the diffuse+glass randomwalk. Hold this for a bit while i add these changes.